packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -28,60 +30,59 @@ import {
 } from '@theia/core/lib/browser';
 import { ContextKeyService } from '@theia/core/lib/browser/context-key-service';
 import { ApplicationShellMouseTracker } from '@theia/core/lib/browser/shell/application-shell-mouse-tracker';
+import { SelectableTreeNode } from '@theia/core/lib/browser/tree/tree-selection';
```

Comment:
It's unfortunate that all the imports in this file are rewriten. It just adds noise to the PR.

Reply:
I also hate that! But somebody need to start applying  the proper imports sort :) 
Okay, I can revert the change.